src/Pages/Video/video-detail.js

```diff
@@ -1,17 +1,21 @@
-import React, { useEffect } from "react";
+import React, { useEffect} from "react";
 import { useAxios } from "../../Hooks/useAxios";
 import { useParams } from "react-router-dom";
 import "./video-detail.css";
 import { BsHeart, BsSuitHeartFill } from "react-icons/bs";
 import { useLiked } from "../../Context/liked-context";
 import { useWatchLater } from "../../Context/watchlater-context";
+import { usePlaylist } from "../../Context/playlist-context";
```

Comment:
put all the context and useContext hook in the index.js file in the context folder, so then it would be more readable